src/layerstack.stanza

```diff
@@ -236,24 +243,25 @@ public defn LayerSpec (
 
 doc: \<DOC>
 Create a Copper layer
```

Comment:
Documenting functions defined in a `#for` like this is difficult. I don't think there is very good support for it yet.